
<header>
<div>
<h1>RECOMB-Seq 2018</h1>
<h2>The Eighth RECOMB Satellite Workshop on Massively Parallel Sequencing</h2>
<h3>Paris, France, April 19-20, 2018 </h3>
</div>
</header>
<div>
<div>
<div>
<h1>RECOMB-Seq<br /> <small>The Eighth RECOMB Satellite Workshop on Massively Parallel Sequencing</small></h1>
</div>
</div>
<div>
<div>
<div>
<div>
<h3>Scope</h3>
<p>A wave of technologies transformed sequencing over a decade ago into the high-throughput era, demanding research in new computational methods to analyze this data. The applications of sequencing have continually expanded. This workshop brings together leading researchers in computational genomics and genomic biology to discuss emerging frontiers in algorithm development for massively parallel sequencing data. The emphasis of all contributed work will be on applying statistical and algorithmic approaches to improve sequence data management, analysis, and interpretation.</p>
</div>
</div>
<div>
<div>
<h3>Contributions</h3>
<p>We solicit contributions in three different categories as follows:</p>
<ol>
<li>
<p>Manuscripts describing original work on computational aspects of genomic research involving massively parallel sequencing.</p>
</li>
<li>
<p>Highlights Track abstracts. 1-2 pages describing computational aspects of work that has been published in a journal after May 1, 2017, or papers that are <em>in press</em> at the time of submission and are already linked on the journal website. This track is joint with the main RECOMB conference.</p>
</li>
<li>
<p>Short talks / Posters / Lightning poster talks</p>
<ul>
<li>
<p>Regular abstracts, 1-2 pages describing original work, including software applications. These will be considered for short presentations as well as for posters.</p>
</li>
<li>
<p>Lightning poster talks will be encouraged to acquaint the audience with work presented in a poster.</p>
</li>
</ul>
</li>
</ol>
</div>
</div>
<div>
<div>
<h3>Topics</h3>
<p>(<em>include but not limited to</em>)</p>
<div>
<div>
<ul>
<li>Read mapping for second and third generation sequencing technologies.</li>
<li>Techniques for managing (e.g., data compression) and visualizing massive amount of sequencing data.</li>
<li>Discovery and genotyping of genomic variants, including SNPs, indels, and structural variants.</li>
<li>Local and de novo sequence assembly.</li>
<li>RNA sequencing (RNAseq) methods, including the analysis of RNA expression and novel transcript assembly.</li>
</ul>
</div>
<div>
<ul>
<li>Methods for emerging sequencing technologies, such as single-cell or single molecule real time sequencing</li>
<li>Translational applications of sequencing data, including cancer genomics and infectious disease diagnostics and surveillance</li>
<li>Epigenetics and gene regulation, including ChIP-seq analysis, methylation profiling, and histone modification</li>
<li>Metagenomics</li>
</ul>
</div>
</div>
</div>
</div>
<div>
<div>
<h3>Paper Submission Procedures</h3>
<p>Manuscripts should be no more than 10 single-spaced US letter or A4 pages with at most a 6.5x9" text area in at least 11 point font. Title and authors, corresponding author's email address, 100-250 word abstract, references, figures, and tables all included. An optional short appendix may contain details or additional data to be consulted at the discretion of the program committee. Manuscripts must be submitted electronically in PDF format via the <a href="https://easychair.org/conferences/?conf=recomb2018" target="_blank">EasyChair system</a>.</p>
<p>Manuscripts submitted for review should represent original, previously unpublished work. At the time of submission to the workshop, and for the entire review period, the work should not be under review by any other conference or scientific journal.</p>
<p>The submission system will be open starting <strong>January 2nd, 2018</strong> and ending <strong> February 4, 2018</strong>. It is required to submit an abstract for your paper before <strong>January 31, 2018</strong>. Authors will be notified of acceptance or rejection by email on or before <strong>February 21th, 2018</strong>. An author of each accepted paper is expected to attend the workshop and present the paper; otherwise, alternative arrangements should be made to have the paper presented.</p>
</div>
</div>
<div>
<div>
<h3><em>Bioinformatics</em> Journal Partners with RECOMB-Seq Satellite Workshop</h3>
<p>The Eighth RECOMB Satellite Workshop on Massively Parallel Sequencing (RECOMB-Seq) is continuing its successful partnership with <em>Bioinformatics</em>. Selected papers accepted for the workshop will be considered for a special track publication in the journal <em>Bioinformatics</em>.</p>
<p><em>Bioinformatics</em> is a biweekly peer-reviewed scientific journal covering research and software in computational biology. The journal was established as Computer Applications in the Biosciences (CABIOS) in 1985. In 1998, the journal obtained its current name. <em>Bioinformatics</em> is published by Oxford University Press, and as of 2016, the editors-in-chief are Alfonso Valencia and Janet Kelso.</p>
</div>
</div>
<div>
<div>
<h3>Special Handling of Papers Submitted to RECOMB 2018</h3>
<p>If you would like to consider submitting a paper rejected by RECOMB 2018 to the satellite workshop, you may do so by the regular deadline of <strong> January 31st</strong>. You must include in your submission a <em>Cover Letter</em> that includes the reviews received from RECOMB together with a rebuttal addressing the concerns raised by the reviewers. The submitted manuscript should also be modified according to the reviews, or provide an explanation of why modifications are unnecessary. All reviews will be treated confidentially and will only be visible to the PC members evaluating your paper.</p>
</div>
</div>
<div>
<div>
<div>
<div>
<div>
<h3></h3>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div>
<div>
<div>
<h3>Schedule At-a-Glance</h3>
<table>
<tbody>
<tr>
<td><strong>Thursday, April 19</strong></td>
</tr>
<tr>
<td>Around 9:00 AM</td>
<td> Registration, start of workshop day 1</td>
</tr>
<tr>
<td>Around 5:30 PM</td>
<td>End of day 1</td>
</tr>
<tr>
<td> </td>
</tr>
<tr>
<td><strong>Friday April 20</strong></td>
</tr>
<tr>
<td>Around 9:00 AM</td>
<td> Registration, start of workshop day 2</td>
</tr>
<tr>
<td>Around 5:30 PM</td>
<td>End of day 2 and workshop</td>
</tr>
</tbody>
</table>
</div>
</div>
<div>
<div>
<h3>Key Dates</h3>
<p><strong>January 31:</strong> Abstract of full paper submission deadline <br /> <strong>February 4 midnight Paris time:</strong> Full paper submission deadline <br /> <strong>February 5:</strong> Reviewers assigned<br /> <strong>February 21:</strong> Reviews due &amp; poster submission deadline<br /> <strong>March 1:</strong> Program announced<br /> <strong>April 19-20:</strong> Conference</p>
</div>
</div>
<div>
<div>
<h3>Invited Speakers</h3>
<p><strong>Alexis Battle</strong><br /> Computational Biology &amp; Medicine Group<br /> Machine Learning &amp; Data Intensive Computing Group<br /> Department of Computer Science<br /> Johns Hopkins, MD, USA </p>
<p><strong>Olivier Jaillon </strong><br /> Tara Oceans Science<br /> Genoscope, Institut de Genomique du CEA, France </p>
</div>
</div>
<div>
<div>
<h3>Organizing Committee</h3>
<p><strong>Program Committee Chairs</strong></p>
<ul>
<li>Rayan Chikhi, University of Lille, France</li>
<li>Mark Chaisson, University of Southern California, USA</li>
</ul>
<p><strong>Steering Committee</strong></p>
<ul>
<li>Inanc Birol, Michael Smith Genome Sciences Centre, Canada</li>
<li>Michael Brudno, University of Toronto, Canada</li>
<li>Eran Halperin, UCLA, USA</li>
<li>Ben Raphael, Princeton University, USA</li>
<li>Cenk Sahinalp, Indiana University, Bloomington, USA</li>
</ul>
<p><strong>Program Committee</strong></p>
<ul>
<li>Can Alkan, Bilkent University</li>
<li>Lars Arvestad, Stockholm University</li>
<li>Sharon Aviran, UC Davis</li>
<li>Niko Beerenwinkel, ETH Zurich</li>
<li>Titus Brown, UC Davis</li>
<li>Mark Chaisson, University of Washington</li>
<li>Ken Chen, MD Anderson</li>
<li>Melissa Gymrek, UCSD</li>
<li>Faraz Hach, Simon Fraser University</li>
<li>Iman Hajirasouliha, Weill Cornell Medical College</li>
<li>Michael Hoffman, Princess Margaret Cancer Centre</li>
<li>Farhad Hormozdiari, Harvard Medical School</li>
<li>Peter Kharchenko, Harvard Medical School</li>
<li>Carl Kingsford, Carnegie Mellon University</li>
<li>Gunnar Klau, VU Amsterdam</li>
<li>Ben Langmead, Johns Hopkins University</li>
<li>Wenxiu Ma, UC Riverside</li>
<li>Veli Makinen, University of Helsinki</li>
<li>Serghei Mangul, UCLA</li>
<li>Tobias Marschall, Max Planck Institute for Informatics</li>
<li>Paul Medvedev, Penn State University</li>
<li>Olgica Milenkovic, University of Illinois at Urbana-Champaign</li>
<li>Niranjan Nagarajan, Genome Institute of Singapore</li>
<li>Robert Patro, SUNY Stony Brook</li>
<li>Teresa Przytycka, National Institutes of Health</li>
<li>Sven Rahmann, University of Duisburg-Essen</li>
<li>Michael Schatz, Johns Hopkins University</li>
<li>Alexander Schliep, Rutgers University</li>
<li>Russell Schwartz, Carnegie Mellon University</li>
<li>Jared Simpson, OICR</li>
<li>Wing-Kin Sung, National University of Singapore</li>
<li>James Taylor, Johns Hopkins University</li>
<li>Fabio Vandin, University of Padova</li>
<li>Jianyang Zeng, Tsinghua University</li>
<li>Shaojie Zhang, University of Central Florida</li>
<li>Fangqing Zhao, Chinese Academy of Sciences</li>
</ul>
</div>
</div>
</div>
</div>
